hack/build.sh

```diff
@@ -2,6 +2,8 @@
 
 set -ex
 
+RHCOS_BUILD_NAME="${RELEASE_BUILD_NAME:-47.330}"
```

Comment:
`/s/RELEASE_BUILD_NAME/RHCOS_BUILD_NAME` ?

Reply:
> `/s/RELEASE_BUILD_NAME/RHCOS_BUILD_NAME` ?

Oops.  I have been making that mistake in released images for a while :/.  Fixed with 6f7c83b2b -> bd88157ab.